### PR TITLE
Certificate section + removed default navbar

### PR DESCRIPTION
**Pull request by 000Andre000**
image
image
image
image
also made connection with graph cms

### CODE REVIEW

**Comment by 000Andre000** [comment]
So i created a model for the hygraph to upload certificates and an id and the site fetches the certificate with the given id

**Comment by 000Andre000** [comment]
About the server to host the node have not done any thing with that its same as the template provided have to see how to do that

**Comment by grejojoby** [comment]
There are few options to host including cloudflare on teknack domain. We can take care of it after dev progress

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,2 @@
+{
```
Comment (grejojoby) [inline .vscode/settings.json:1]: Dont commit .vscode files. The folder should be in gitignore

**Review by grejojoby — APPROVED** [review]

**Pull request merged**